src/SFA.DAS.Forecasting.Database/Tables/Payment.sql

```diff
@@ -3,6 +3,7 @@
 	[Id] BIGINT NOT NULL PRIMARY KEY Identity(1,1),
 	[ExternalPaymentId] NVARCHAR (100) NOT NULL,
 	[EmployerAccountId] BIGINT NOT NULL,
+	[SendingEmployerAccountId] BIGINT NOT NULL default(0),
```

Comment:
May have missed this one but wont this always be populated, with either the account id its being sent to or your employer account id? If so I'd be tempted to remove the default value

Reply:
There will already be existing data in this table so we'll need to set a default value for those rows.